src/window.py

```diff
@@ -225,6 +228,11 @@ def _populate(self, directory):
         self._worker.connect("failed", self._on_load_failed)
         self._worker.start()
 
+        if self._bookmarks.is_bookmarked(directory):
```

Comment:
I think we should have a custom button class, so we can move this kind of things (and other specific loggic) inside.